### ISSUE TITLE
After an NVDA update, a new copy cannot be started and a dialog box with an error message appears

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#12574]
### Steps to reproduce:
1. Install or create a portable version of NVDA2021.1Beta3.
2. Start NVDA2021.1Beta3 and try to check for updates under the help menu.
3. It is detected that NVDA2020.1Beta4 is available, and it is updated according to the normal process.
### Actual behavior:
When you complete step (4), You will hear the prompt sound to start NVDA, but you will not enter the new version update process, The new version is not installed.
When you open Narrator or other screen readers, you will find an error dialog box with the following message:
Error usage: nvda_noUIAccess.exe [-h] [-q] [-k] [-f LOGFILENAME]
[-l {10,12,15,20,30,40,50,100}] [-c CONFIGPATH]
[-m] [-s] [--disable-addons] [--debug-logging]
[--no-logging] [--no-sr-flag]
[--install | --install-silent | --create-portable | --create-portable-silent]
[--portable-path PORTABLEPATH] [--launcher]
[--enable-start-on-logon True|False]
[--copy-portable-config] [--ease-of-access]
error: argument -m/--minimal: ignored explicit argument '--log-level=10'
 
### Expected behavior:
NVDA will be updated, And no errors

#### NVDA installed/portable/running from source:
installed and portable
#### NVDA version:
NVDA2021.1Beta3

#### Windows version:
Windows 10 21H1 (64位) build 19043.1055

#### Name and version of other software in use when reproducing the issue:
none

#### Other information about your system:
none

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
This problem also occurs when updating from the penultimate alpha version to the latest alpha version.
I suspect this is a regression of #12562 

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Brian1Gaff** [comment]
Yes this version firstly is running the dropping tones at the same time as 
its running the start up jingle of the new version and then it all goes 
completely silent and you have to manually reboot a known working version. 
Here is the error on a protable update.

usage: nvda_noUIAccess.exe [-h] [-q] [-k] [-f LOGFILENAME]
                           [-l {10,12,15,20,30,40,50,100}] [-c CONFIGPATH]
                           [-m] [-s] [--disable-addons] [--debug-logging]
                           [--no-logging] [--no-sr-flag]
                           [--install | --install-silent | --create-portable 
| --create-portable-silent]
                           [--portable-path PORTABLEPATH] [--launcher]
                           [--enable-start-on-logon True|False]
                           [--copy-portable-config] [--ease-of-access]

error: argument -m/--minimal: ignored explicit argument '--log-level=10'

Means nothing to me, hopefully it will to somebody!

Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the

**Comment by feerrenrut** [comment]
From the log:
```
DEBUG - core._startNewInstance (17:24:19.599) - MainThread (20144):
Starting new NVDA instance: NewNVDAInstance(filePath='C:\\Users\\user\\AppData\\Local\\Temp\\nvda_update_l4ri0_o5.exe', parameters='--install -m--log-level=10', directory=None)
```

**Comment by XLTechie** [comment]
I can confirm that this, or something very similar to it, happened to me 
yesterday while updating from the try-build of #12560 which was based on 
recent beta branch, to a normal alpha.

**Comment by michaelDCurran** [comment]
Line 223 of updateCheck.py:
```
executeParams += f"--log-level={log.level}"
```

That string needs to start with a space.

**Comment by michaelDCurran** [comment]
Note that there will be no way to fix this for anyone running the affected alphas / beta3 or beta4. They will have to install a fixed copy manually when it is out.

**Comment by leonardder** [comment]
Fixed in #12575
